src/js/footer.js

```diff
@@ -0,0 +1,61 @@
+import Toggle from 'o-toggle';
+import layout from './layout';
+
+const COLLAPSIBLE_BREAKPOINTS = ['default', 'XS', 'S'];
+
+class Footer {
+
+	constructor (footerEl) {
+		if (!footerEl) {
+			footerEl = document.querySelector('[data-o-component="o-footer"]');
+		} else if (typeof footerEl === 'string') {
+			footerEl = document.querySelector(footerEl);
+		}
+
+		this.footerEl = footerEl;
+
+		layout(breakpoint => {
+			if (COLLAPSIBLE_BREAKPOINTS.indexOf(breakpoint) !== -1 && !this._toggles) {
```

Comment:
The heading element is the toggle so it's always visible - but what I primarily wanted to avoid was having all the unnecessary aria state applied for larger screens which is confusing.